action.yml

```diff
@@ -221,4 +221,9 @@ runs:
           aws cloudfront create-invalidation --distribution-id ${{ inputs.aws-cloudfront-distribution-id }} --paths "/${{ steps.variables.outputs.siteId }}/${{ steps.variables.outputs.branch }}/*"
           if [ -f tests/lighthouse/runs/manifest.json ]; then aws cloudfront create-invalidation --distribution-id ${{inputs.aws-cloudfront-distribution-id }} --paths "/${{ steps.variables.outputs.siteId }}/.lighthouse/${{ steps.variables.outputs.branch }}/manifest.json"; fi;
         fi
-        
\ No newline at end of file
+
+    - run: date -u +"%FT%TZ" | tr -d '\n' > date.txt && npx -y jjo timestamp=@date.txt type=snap-action data=$(npx -y jjo repository=${{ steps.variables.outputs.repository }} conclusion=${{ github.action_status }}) > SnapAction\[${{ steps.variables.outputs.repository }}\]-$(date -u +"%Y_%m_%d_%H%M").json && aws s3 cp --region us-east-1 ./SnapAction*.json s3://datawarehouse-stage/team-metrics-import/
```

Comment:
I'm guessing this falling outside of a named step with the `if` is what will cause this to run on both success and failure?